core/bind/core_bind.cpp

```diff
@@ -562,6 +562,14 @@ String _OS::get_locale() const {
 	return OS::get_singleton()->get_locale();
 }
 
+String _OS::get_standardized_locale(bool p_posix_locale) const {
+
+	if (p_posix_locale)
+		return get_locale().replace("-", "_");
+
+	return get_locale().replace("_", "-");
```

Comment:
No, what I meant was that the method should return the standard that Godot follows (which is glibc's).

Reply:
would this be ok?
```c++
String _OS::get_standardized_locale(bool p_posix_locale) const {
	return get_locale().replace("-", "_");
}
```
@akien-mga

Reply:
Yeah. Be sure to add the matching documentation too.